examples/exampleGuide.md

```diff
@@ -28,8 +28,8 @@ More [Sample templates](https://github.com/Azure/azure-quickstart-templates/tree
     scope: managementgroup 
     managementGroupId: demoId 
     region: centralus 
-    template: https://teststorage.blob.......... /template.json
-    parameters: https://teststorage.blob.......... /parameters.json
+    template: https://teststorage.blob/template.json

```

Comment:
Can't we move these templates to this repo itself?

Reply:
This is intentional to show that any external link can be used.